profile_bluesky/startup/instrument/plans/energy_plans.py

```diff
@@ -2,14 +2,15 @@
 Slitscan
 """
 
-__all__ = ['moveE']
+__all__ = ['moveE','Escan','Escan_list','qxscan']
```

Comment:
Add whitespace between these words.  Easier to read (and debug) and we are not trying to fit this onto punch cards anymore.

Reply:
consider using pylint, flake8, black or some other code format checker